index.html

```diff
@@ -31,7 +31,167 @@ <h1>Charlie Giorgania</h1>
         <img class="base" src="./images/base.png" alt="" />
       </div>
 
-      <div></div>
+      <section class="projectCards">
+        <div>
```

Comment:
[OPTIONAL]
- [ ] The main container of the projects should be aligned with the `display: grid` property. Kindly fix that.